Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2536 -- rename `/policy-repository/` to `/subjects/`

Resolves [EREGCSC-2536](https://jiraent.cms.gov/browse/EREGCSC-2536)

**Description**

We are renaming the "Policy Repository" name to be "Find by Subject".

This includes changing the URL path from `/policy-repository/` to `/subjects/`

**This pull request changes:**

- Removes `policy-repository/search/` page entirely
- Renames directories, filenames, variables, CSS classes, etc away from Policy Repository and closer to Subjects as appropriate
- Adds `/subjects/` URL in Django
- Keeps the `/policy-repository/` url and Django view in order to redirect visitors to the Subject Browser if they are using a saved/bookmarked `/policy-repository/` URL.
    - This redirect should keep URL parameters intact
 - Updates unit and e2e tests
 - Adds e2e test to verify the redirect

**Steps to manually verify this change:**

1. Visit the experimental deployment using a `/policy-repository` URL and make sure it reliably redirects to `/subjects` with URL parameters intact
   - ex: https://p1wlrprv0c.execute-api.us-east-1.amazonaws.com/dev1169/policy-repository?q=test&subjects=2
1. Poke around [subjects page](https://p1wlrprv0c.execute-api.us-east-1.amazonaws.com/dev1169/subjects) and make sure things work as expected
